tests/integration/test_package_managers.py

```diff
@@ -123,7 +123,7 @@ def test_packages(
         cmd += test_params.flags
 
     for package in test_params.packages:
-        cmd += ["--package", json.dumps(package).encode("utf-8")]
+        cmd += [json.dumps(package).encode("utf-8")]
```

Comment:
We don't accept the input JSON more than once, so this is not correct (should we?)

Reply:
I don't think we should. I fixed this line.